config/secrets.yml

```diff
@@ -81,6 +81,7 @@ development:
   from_email: Brave Payments Publishers Dev<brave-publishers@localhost.local>
   attr_encrypted_key: 499a4c51df667b4edfab40c1f8b813b7ed6ce02096d59f23b5dcb095369375f6
   secret_key_base: dd7b12788a804315fd75f1ff97fae33310b451c39d83d1d015543d25d0ba034c02fd312e83735f0d0aeaaf70131f26e614629e3c40531b949445b4dfacd3bad3
+  uphold_authorization_endpoint: "https://uphold.example.com/authorize/<UPHOLD_CLIENT_ID>?scope=<UPHOLD_SCOPE>&intention=signup&state=<STATE>"
```

Comment:
I got an error on Publishers#home, I think we need to add
```yml
uphold_client_id: test_client_id
uphold_client_secret: test_client_secret
uphold_scope: a:read,b:read,c:deposit
```
to development like we did with test.

Reply:
good catch. i updated it and tested out Publishers #home